ts/src/renderer/phaser/classes/PhaserEntity.ts

```diff
@@ -0,0 +1,61 @@
+abstract class PhaserEntity extends Phaser.GameObjects.Container {
+
+	sprite: Phaser.GameObjects.Sprite;
+    key: any;
+
+	private playAnimationListener: EvtListener;
```

Comment:
do all entities contain `sprite` and are able to play animations? I can think of region and chat bubble not having this logic. we should probably have a `PhaserAnimatedEntity` class that extends `PhaserEntity` and adds sprite and animation listener, so that entities that need it can extend animated entity and others base entity.

Reply:
good idea

Reply:
I know, just giving suggestions